src/network/networkd-network.h

```diff
@@ -164,6 +164,7 @@ struct Network {
         bool allow_port_to_be_root;
         bool unicast_flood;
         unsigned cost;
+        unsigned priority;
```

Comment:
.... and despite netlink type uint16_t, the actual bridge port priority valid values are 0..63. And brctl manpage is a lie.